client/src/constants/index.ts

```diff
@@ -0,0 +1,3 @@
+export enum Constants {
+  BASE_URL = 'http://localhost:3000'
+}
```

Comment:
no need to store base url as enum, simple variable in current file will be enough